Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/entrycard.vue

```diff
@@ -84,6 +100,20 @@ export default class EntrycardTimelineComponent extends Vue {
                             }}</slot>
                         </b-col>
                         <b-col cols="4" class="text-right align-self-center">
+                            <span
+                                v-if="commentsCount > 1"
+                                class="pr-2"
+                                data-testid="commentCount"
+                                >{{
+                                    commentsCount > 1 ? commentsCount : ""
```

Comment:
this is no longer needed since the section only renders if there are more than one comment

Reply:
aha nice , I forgot to remove the 2nd one :)

Reply:
code updated and tested